modules/installation-restricted-network-samples.adoc

```diff
@@ -29,6 +32,11 @@ ifdef::samplesoperatoraltreg[]
 Operator.
 endif::[]
 
+Setting the registry override field in the Sample Operator configuration file to link:registry.redhat.io[registry.redhat.io] is redundant because it is already directed to link:registry.redhat.io[registry.redhat.io] for everything but Jenkins images and imagestreams. It also breaks the installation payload for Jenkins imagestreams.
```

Comment:
Do we define the `registry override` field? Is that the same as the `samplesRegistry` field in Step 4?

Reply:
IMO, the `registry override` is meaning `samplesRegistry`.
Only `samplesRegistry` in samples operator configuration does this function.

Reply:
Thank you for pointing that out! I'm updating it now.